crates/proc-macro-srv/src/lib.rs

```diff
@@ -11,11 +11,12 @@
 //!   rustc rather than `unstable`. (Although in general ABI compatibility is still an issue)…
 
 #![cfg(any(feature = "sysroot-abi", rust_analyzer))]
-#![feature(proc_macro_internals, proc_macro_diagnostic, proc_macro_span)]
+#![feature(proc_macro_internals, proc_macro_diagnostic, proc_macro_span, rustc_private)]
```

Comment:
This feature needs to be gated `#[cfg_attr(feature = "in-rust-tree", feature(rustc_private)]`